src/utils/index.js

```diff
@@ -0,0 +1,17 @@
+import { Random } from '@woowacourse/mission-utils';
+import { LOTTO } from '../constants/index.js';
+
+const calculateQuatity = (purchaseAmount) =>
+  parseInt(purchaseAmount, 10) / LOTTO.UNIT_PRICE;
+
+const generateRandomNumbers = () =>
+  Random.pickUniqueNumbersInRange(
+    LOTTO.MINIMUM_NUMBER,
+    LOTTO.MAXIMUM_NUMBER,
+    LOTTO.NUMBER_OF_SPACE,
+  );
```

Comment:
```suggestion
const generateRandomNumbers = (minRange, maxRange) =>
  Random.pickUniqueNumbersInRange(
    minRange,
    maxRange,
    LOTTO.NUMBER_OF_SPACE,
  );
```
함수 이름이 `generateRandomNumbers`로 로또 번호에 한정적이지 않아서 랜덤 숫자의 범위도 인자로 받으면 더 범용적인 함수로 사용할 수 있을 것 같아요~!

Reply:
좋은 의견인 것 같습니다!🙂